l2-contracts/contracts/bridge/L2StandardERC20.sol

```diff
@@ -37,8 +38,8 @@ contract L2StandardERC20 is ERC20PermitUpgradeable, IL2StandardToken, ERC1967Upg
     /// @dev Address of the L1 token that can be deposited to mint this L2 token
     address public override l1Address;
 
-    modifier onlyBridge() {
-        if (msg.sender != l2Bridge) {
+    modifier onlyNTV() {
```

Comment:
could you please add a comment on deprecation for `address public override l2Bridge` above?